base/broadcast.jl

```diff
@@ -49,6 +52,8 @@ broadcast_indices(A) = broadcast_indices(containertype(A), A)
 broadcast_indices(::Type{Any}, A) = ()
 broadcast_indices(::Type{Tuple}, A) = (OneTo(length(A)),)
 broadcast_indices(::Type{Array}, A) = indices(A)
+broadcast_indices(::Type{Nullable}, x) =
+    ifelse(isnull(x), (Nullable(1),), (Nullable{Int}()))
```

Comment:
More simply written as `(Nullable(1, !isnull(x)),)`. BTW, the second part of the `ifelse` does not return a tuple.